src/project-tests/markdown-previewer-tests.js

```diff
@@ -309,7 +309,24 @@ export default function createMarkdownPreviewerTests() {
         // are actually the ones represented by the markdown:
 
         // find matching H1 element
-        h1Text = (/#\s.*/).exec(markdown)[0].slice(2);
+
+        // Added the m modifier to match hashes at the beginning of a paragraph
+        // so that people can use hashes for h(n>2) ATX headings in addition to
+        // their Setext headings
```

Comment:
What does using hashes for a heading look like? What is an ATX heading? What is a Setext heading?

If there is a reference document I should be familiar with first, it would also be ok to mention it at the top of this file rather than in this comment.